src/foam/u2/view/ColumnConfigView.js

```diff
@@ -599,6 +611,7 @@ foam.CLASS({
                .end()
                .start()
                 .style({'padding-left' : '8px'})
```

Comment:
Can you add this CSS to the css axiom as well since we have a class for this div now

Reply:
ok